pkg/proc/interface.go

```diff
@@ -70,6 +66,10 @@ type Info interface {
 	BinInfo() *BinaryInfo
 	EntryPoint() (uint64, error)
 
+	AdjustsPCAfterBreakpoint() bool
```

Comment:
IMHO this is evidence that the code that this PR is unifying isn't actually common among different backends and shouldn't be unified.

I also don't think that having a variable called adjustPC and a method called AdjustsPCAfterBreakpoint have opposite meanings (AdjustsPCAfterBreakpoint returning true means we set adjustPC to false) is a good idea.

Reply:
I disagree, I don't think it's a problem to "ask" the backend something, this is a simple predicate method. I don't think it's worth duplication of code just to prevent asking a single question to determine whether we should do some arithmetic on a pointer.

Reply:
I'm voting for leaving it as a method for now, unless you really object. I think it's a cleaner solution instead of adding another config param to NewTarget.

I'm open to other names for this method as well. My thought process on the current name is that we are asking the backend if it adjusts the pc after hitting a breakpoint, so that's why I phrased it as I did. Setting `adjustPC` to true when the method call is false is saying "we must adjust the PC if the backend does not".

Reply:
Let's call it AdjustPC and have it match the direction of adjustPC.